packages/core/src/tooltip/Tooltip.tsx

```diff
@@ -89,7 +89,8 @@ export const Tooltip = forwardRef<HTMLDivElement, TooltipProps>(
       validationStatus: formFieldValidationStatus,
     } = useFormFieldProps();
 
-    const disabled = formFieldDisabled ?? disabledProp;
+    const disabled = Boolean(formFieldDisabled) || disabledProp;
```

Comment:
Can you add a story or a test to show the bug? I tried 
`
  <FormField disabled>
    <Tooltip {...props}>
      <Button>Hover</Button>
    </Tooltip>
  </FormField>
`
and it doesn't show the tooltip.

Reply:
Is this expected? disabled tooltip won't show?

Reply:
do you need to display a tooltip even if no truncation?

Reply:
exactly

Reply:
Ultimately the logic here is wrong, with two booleans they should be "OR"d but in the case it's using the undefined check